lib/vertica/query.rb

```diff
@@ -83,7 +83,8 @@ def handle_data_row(message)
   def handle_command_complete(message)
     if buffer_rows?
       @result = Vertica::Result.new(row_description: @row_description, rows: @buffer, tag: message.tag)
-      @row_description, @buffer = nil, nil
+      @row_description = nil
+      @buffer.clear if buffer_rows?
```

Comment:
I just noticed that we're already in an `if buffer_rows?` block (see line 83). I can we can just always just clear it here and get rid of the if suffix.